lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -28,8 +28,8 @@
     <datatype extension="anvio_variability" type="galaxy.datatypes.tabular:TSV" display_in_upload="false" subclass="true" />
     <datatype extension="arff" type="galaxy.datatypes.text:Arff" mimetype="text/plain" display_in_upload="true"/>
     <datatype extension="paf" auto_compressed_types="gz" type="galaxy.datatypes.text:Paf" mimetype="text/plain" display_in_upload="true"/>
-    <datatype extension="gfa1" type="galaxy.datatypes.text:Gfa1" mimetype="text/plain" display_in_upload="true"/>
-    <datatype extension="gfa2" type="galaxy.datatypes.text:Gfa2" mimetype="text/plain" display_in_upload="true"/>
+    <datatype extension="gfa1" auto_compressed_types="gz" type="galaxy.datatypes.text:Gfa1" mimetype="text/plain" display_in_upload="true"/>
+    <datatype extension="gfa2" auto_compressed_types="gz" type="galaxy.datatypes.text:Gfa2" mimetype="text/plain" display_in_upload="true"/>
```

Comment:
Do tools that consume gfa files expect them to be tabix-indexable if they're gz-compressed ? Can you also add a compressed copy of the two gfa files in lib/galaxy/datatypes/test ?

Reply:
Sure, thanks for reviewing. I'll ask about the indexation.

Reply:
I asked the developer of gfastats and he told me that it is not necessary to be tabix-indexable.